observation/interferometric_pointing.py

```diff
@@ -87,7 +88,6 @@ class NoTargetsUpError(Exception):
                                 session.ants.req.offset_fixed(offset_target[0], offset_target[1], opts.projection)
                             nd_params = session.nd_params
                             #session.fire_noise_diode(announce=True, **nd_params)
-                            target.tags = target.tags[:1]  # this is to avoid overloading the cal pipeline
```

Comment:
There was some rationale behind this... Please see the original PR #385.

Reply:
i have reviewed the comments in PR #385  and will revert this change.

however take note of my reason for wanting to do this. without this change, pointing datasets result in MANY MANY of the following warnings being logged when the datasets are loaded and also when processed:

> WARNING:katpoint.catalogue:Found different targets with same name(s) 'J1256-0547, 1253-055, 3C279' in catalogue

it should not be acceptable to have to ignore warnings as a rule.

perhaps i should propose a change to katpoint.catalogue, to not log warnings for targets that differ solely in "extra tags"?

Reply:
it seems like i am making spaghetti of the review comment threads...

my latest commit implements the suggestion by @spassmoor and @ludwigschwardt to scan along a circle around bore sight. it also happens to remove the deepest nesting from the code and also shortens the nested code by 5 lines.